harvester/sources/tests/extraction/test_edurep.py

```diff
@@ -0,0 +1,266 @@
+            else:
+                self.assertEqual(state, "active")
+
+    def test_lowest_education_level(self):
+        seeds = self.seeds
+
+        self.assertEqual(seeds[0]["lowest_educational_level"], 1,
+                         "Expected file with MBO to have lowest educational level 1")
+        self.assertEqual(seeds[1]["lowest_educational_level"], 1,
+                         "Expected file with MBO to have lowest educational level 1")
+        self.assertEqual(seeds[2]["lowest_educational_level"], 2,
+                         "Expected file with HBO to have lowest educational level 2")
+
+    def test_keywords(self):
+        seeds = self.seeds
+        self.assertTrue("asperge" in seeds[0]["keywords"])
+        self.assertTrue("wortelen" in seeds[0]["keywords"])
+        self.assertTrue("vollegrondsgroenten" in seeds[0]["keywords"])
+        self.assertTrue("sperziebonen" in seeds[0]["keywords"])
+
+    def test_copyright_description(self):
+        seeds = self.seeds
+        self.assertEqual(seeds[2]["copyright_description"], "cc-by-sa-30")
+        self.assertEqual(seeds[1]["copyright_description"], "cc-by-30")
+        self.assertEqual(seeds[0]["copyright_description"], "cc-by-30")
+        self.assertEqual(seeds[3]["copyright_description"], "http://creativecommons.org/licenses/by-nc-sa/4.0/")
+        self.assertEqual(seeds[4]["copyright_description"], "onbekend")
+        self.assertEqual(seeds[5]["copyright_description"], "cc-by-nc-nd-30")
+
+    def test_copyright(self):
+        seeds = self.seeds
+        self.assertEqual(seeds[2]["copyright"], "https://creativecommons.org/licenses/by-sa/3.0/nl/")
+        self.assertEqual(seeds[3]["copyright"], "cc-by-nc-sa-40")
+        self.assertEqual(seeds[4]["copyright"], "yes")
+        self.assertEqual(seeds[5]["copyright"], "https://creativecommons.org/licenses/by-nc-nd/3.0/nl/")
+
+    def test_material_types(self):
+        seeds = self.seeds
+
+        # wikiwijsmaken
+        self.assertTrue("handleiding" in seeds[0]["material_types"])
+        self.assertTrue("open opdracht" in seeds[0]["material_types"])
+        self.assertTrue("gesloten opdracht" in seeds[0]["material_types"],
+                        f"gesloten opdracht not in {seeds[3]['material_types']}")
+
+        # l4l
+        self.assertTrue("informatiebron" in seeds[3]["material_types"],
+                        f"informatiebron not in {seeds[3]['material_types']}")
+        self.assertEqual(len(seeds[3]["material_types"]), 1)
+
+        # WikiwijsDelen
+        self.assertTrue("informatiebron" in seeds[4]["material_types"])
+        self.assertTrue("informatiebron" in seeds[5]["material_types"])
+
+    def test_analysis_allowed(self):
+        seeds = self.seeds
+        self.assertEqual(seeds[2]["analysis_allowed"], True,
+                         "Expected OpenAccess file to allow analysis")
+        self.assertEqual(seeds[4]["analysis_allowed"], False, f"analysis is {seeds[4]['analysis_allowed']}")
```

Comment:
@fako  Should the middle assertEqual be false? I was not sure why some AccessRestricted files would be true and some false.